### ISSUE TITLE
Szövegkorrekció: Morgath Birodalom => Morgath Konföderáció

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#816]
```
Mint a Dribeknél, itt is birodalom van írva.
```

Original issue reported on code.google.com by `kli...@gmail.com` on 18 Jan 2014 at 8:29

**Comment by GoogleCodeExporter** [comment]
```
Javítva. A diplomáciában 1 helyen is birodalomként szerepelt, de mindenhol 
máshol meg konföderációként.
```

Original comment by `akarn...@gmail.com` on 21 Feb 2014 at 8:08
- Changed state: **SVN**
- Added labels: **Priority-Low**, **Labels**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.95.163
```

Original comment by `akarn...@gmail.com` on 21 Feb 2014 at 10:59
- Changed state: **Fixed**
